application/baremetal/benchmark/coremark/toolchain_terapines.mk

```diff
@@ -1 +1 @@
-BENCH_FLAGS ?= -O3 -flto
+BENCH_FLAGS ?= -O3 -flto -falign-functions=4 -falign-loops=4 -flate-loop-unroll -malign-branch -Wl,-mllvm,--align-all-nofallthru-blocks=2 -Wno-unused-command-line-argument
```

Comment:
Can you remove `-Wno-unused-command-line-argument`  and remove the unused options not supported by Zcc.

Reply:
"-Wl,-mllvm,--align-all-nofallthru-blocks=2" is only used in the linking phase and will raise a warning in the compilation phase. "-Wno-unused-command-line-argument" is used to dismiss this warning.
Or I can use 
```
BENCH_FLAGS ?= -O3 -flto -falign-functions=4 -falign-loops=4 -flate-loop-unroll -malign-branch
LDFLAGS += -Wl,-mllvm,--align-all-nofallthru-blocks=2
```
to replace it?